roles/patch/tasks/patch_vars.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Patch | include defaults
+  include_vars:
+    file: roles/common/defaults/main.yml
+
+- name: Patch | Get the highest version number from gi_patches array
+  set_fact:
+    # Using expression from bms-toolkit/roles/rac-db-setup/tasks/rac-db-install.yml Line 46
+    highest_version_num: "{% if not ansible_loop.first %}{% if item is version(highest_version_num, '>=') %}{{ item }}{% else %}{{ highest_version_num }}{% endif %}{% else %}{{ item }}{% endif %}"
+  loop: "{{ gi_patches | json_query('[?base==`' + oracle_ver + '`].release') }}"
+  loop_control:
+    extended: yes
+  when:
```

Comment:
For consistency with existing code, can we add a "tag: patch_vars" to each task here?  It allows for better re-use in the future, where steps can be given multiple tags, and an easy way to enable/disable tasks without too many when clauses etc.

Reply:
Done